src/build.rs

```diff
@@ -136,9 +137,7 @@ impl<'a, const VERBOSE: bool> Resolver<'a, VERBOSE> {
     where
         F: FnOnce() -> String,
     {
-        if VERBOSE {
```

Comment:
This was actually using generics to create two versions, one with the printlns disabled. that did save a bunch of time when I tested it.